### ISSUE TITLE
Error when csv file contains escape quotes.

### ISSUE THREAD

**Issue by huhudage** [geoffleyland/lua-csv#8]
if csv file contains escape quotes, such as
`key1, "a\"b\"c"`

csv.lua will throw "unmatched quote" exception.

**Comment by SiENcE** [comment]
Yes, same to me. this should be ignored...i don't know why this is checked and an exception is thrown.

i think this code needs a rework.

https://github.com/geoffleyland/lua-csv/blob/master/lua/csv.lua#L380-L383
